File: src/main/java/com/summer/lijiahao/utils/OpenNccdevAction.java
package com.summer.lijiahao.utils;

import com.intellij.ide.BrowserUtil;
import com.intellij.openapi.actionSystem.AnActionEvent;
import com.summer.lijiahao.abs.AbstractAnAction;

public class OpenNccdevAction extends AbstractAnAction {
    private final static String DEVURL = "https://nccdev.yonyou.com/index";

    @Override
    public void doAction(AnActionEvent event) {
        BrowserUtil.open(DEVURL);
    }
}

File: src/main/kotlin/com/summer/lijiahao/nccdevtool/menu/action/OpenNccdevAction.kt
package com.summer.lijiahao.nccdevtool.menu.action

import com.intellij.ide.BrowserUtil
import com.intellij.openapi.actionSystem.AnActionEvent
import com.summer.lijiahao.nccdevtool.base.action.AbstractAnAction

class OpenNccdevAction : AbstractAnAction() {
    private val DEVURL = "https://nccdev.yonyou.com/index"
    override fun doAction(event: AnActionEvent) {
        BrowserUtil.open(DEVURL)
    }
}
